spec/system/visit_user_journey_confirmation_page_spec.rb

```diff
@@ -53,8 +79,8 @@
       certificate = create(:sp_signing_certificate)
       sp_component = certificate.component
       visit confirmation_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
-      expect(page).to have_content 'VSP'
-      expect(page).to have_content 'delete the old signing key and certificate from your VSP configuration'
+      expect(page).to have_content 'SP'
+      expect(page).to have_content 'delete the old signing key and certificate from your SP configuration'
```

Comment:
To save further breaking changes to tests if content is changed, it would be good to use the localisation `t` method when we can going forward.

Reply:
Would it be better to use the constants? e.g `COMPONENT_TYPE::SP_SHORT`

Reply:
Totes in this instance 👍